docs/common-issues.md

```diff
@@ -22,6 +22,17 @@ why I'm always positive:
 
     Thanks!
     ```
+    
+### Student asks for extension
+
+If a student asks for an extension on a task, you can go ahead and give it to them 
+as long as they have *some* interaction on the task. The deadlines on tasks are 
+mostly to allow tasks to go back into the pool if a student isn’t actively working 
+on it, rather than to be a difficult deadline to meet.
+```
+I've given you an extra day to complete the task, since you seem to be 
+diligently working on it.
```

Comment:
Maybe we want the default extension to be a couple days?  I feel like on a base of 3-7 days, adding 1 day is asking for needing to do it multiple times.